vehicle-simulator/deployment.json

```diff
@@ -0,0 +1,32 @@
+{
+    "container_name": "vehicle-simulator",
+    "image": {
+        "name": "<image:version>"
+    },
+    "config": {
+      "env": [
+          "ADAPTER_BROKER_ADDRESS=ctrhost"
+      ],
+      "cmd": [
+        "/bin/app",
+        "--ctd-effect-delay=30",
+        "--periodic-updates-delay=5",
+        "--idle-state-time=120"
+      ]
+    },
+    "host_config": {
+      "privileged": true,
+      "extra_hosts": [
+          "ctrhost:host_ip"
+      ],
+      "log_config": {
+        "driver_config": {
+          "type": "json-file",
+          "max_files": 2,
+          "max_size": "100M",
+          "root_dir": "/home/user/log/vehicle-simulator"
+        }
+      }
```

Comment:
Maybe we can remove that. The difference from the default config is the _root_dir_. However the _/home/user/log/vehicle-simulator_ path seems to not be universal enough. I would assume that an user named _`user`_ is needed.